CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v1.3.0
+
+## 04/29/2020
+
+- Migrated from using CARTO tables for VIIRS data to the Elasticsearch datasets generated by geotrellis.
+- Removed all download link options except csv, json and geojson since these can't be supported with datasets. kml/shp/svg download links will no longer be available.
+- Added staging datasets, but production datasets don't exist yet. REMEMBER: add production datasets to config before pushing to production.
```

Comment:
The version number should reflect state in production, so better make the comment somewhere else, ie JIRA